nuxeo-java-client-test/src/test/java/org/nuxeo/client/ITRepository.java

```diff
@@ -695,4 +694,37 @@ private String buildErrorMsgForDate(String key, Class<?> valueType) {
                 key, valueType.getTypeName());
     }
 
+    @Test
+    public void itCanManageAnnotations() {
+        assumeTrue("itCanManageAnnotations works only since Nuxeo 10.2",
+                // TODO change the version to LTS
```

Comment:
remove the TODO or associate with an open JIRA ticket

Reply:
I won't, for each LTS release we need to do manual tasks on this client for tests, constants and this kind of stuff.